promptulate/llms/zhipu/platform.py

```diff
@@ -0,0 +1,247 @@
+import json
+import time
+from abc import ABC
+from json import JSONDecodeError
+from typing import Any, Dict, Iterator, List, Optional, TypeVar, Union
+
+import jwt
+import requests
+from pydantic import BaseModel
+
+from promptulate.config import pne_config
+from promptulate.error import NetWorkError
+from promptulate.llms import BaseLLM
+from promptulate.preset_roles.prompt import PRESET_SYSTEM_PROMPT_ZhiPU
+from promptulate.schema import (
+    AssistantMessage,
+    BaseMessage,
+    LLMType,
+    MessageSet,
+    SystemMessage,
+    UserMessage,
+)
+from promptulate.utils import logger
+
+T = TypeVar("T", bound=BaseModel)
+
+
+class ZhiPuStreamIterator:
```

Comment:
We aim to develop a versatile Pne StreamIterator, designed to encapsulate the streaming responses from various models. Unlike the `ZhiPuStreamIterator`, which is tailored for ZhiPu-type data, or those designed for OpenAI-type data, the unique StreamIterator will be agnostic to the data source. Like `BaseMessage` type, ensuring compatibility and flexibility across different model outputs.

This revised text conveys the intention to create a flexible iterator capable of handling a standardized message format, making it adaptable to multiple sources of streamed data.